apps/condo/domains/common/utils/moneyToWords/locales/en.spec.js

```diff
@@ -0,0 +1,77 @@
+
+const { cloneDeep } = require('lodash')
+
+const { moneyToWords } = require('../moneyToWords')
+
+const locale = 'en'
+const currencyCode = 'USD'
+
+const testIntegers = [
+    [0, 'zero'],
+    [51, 'fifty one'],
+    [137, 'one hundred thirty seven'],
+    [700, 'seven hundred'],
+    [1100, 'one thousand one hundred'],
+    [4680, 'four thousand six hundred eighty'],
+    [63892, 'sixty three thousand eight hundred ninety two'],
+    [86100, 'eighty six thousand one hundred'],
+    [792581, 'seven hundred ninety two thousand five hundred eighty one'],
+    [2741034, 'two million seven hundred forty one thousand thirty four'],
+    [86429753, 'eighty six million four hundred twenty nine thousand seven hundred fifty three'],
+]
+
+const testNegativeIntegers = cloneDeep(testIntegers)
+testNegativeIntegers.map((row, i) => {
```

Comment:
map already returns you a new object, is it not? 
![image](https://github.com/user-attachments/assets/41d2b7b4-3488-4af8-b9e6-0988f1b4550a)

Reply:
Why do you need cloneDeep here?